website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md

```diff
@@ -1,166 +1,81 @@
-
-   Set the values in the `applicationConfig` section:
-
-   ```yaml
-     APPSMITH_ENCRYPTION_PASSWORD: "<PASSWORD>"
-     APPSMITH_ENCRYPTION_SALT: "<SALT>"
-   ```
-
-## Uninstall Community Edition Helm chart
-
-To uninstall the CE helm chart, run the following command:
-
-```bash
-helm uninstall appsmith
-```
-
-## Configure parameters
-
-To ensure that the Appsmith Business Edition Helm chart runs, you need to make some changes to the `values.yaml` file. Follow the steps below to configure parameters:
-
-1. Add the following snippet to the bottom of your `values.yaml` file. This snippet enables PostgreSQL.
-
-   ```yaml
-   postgresql:
-     enabled: true
-     auth:
-       username: root
-       password: "<PASSWORD>"
-       postgresPassword: "<POSTGRESQL_PASSWORD>"
-       database: keycloak
-   ```
-
-2. If present, remove the highlighted lines, regardless of their value.
-
-   ```yaml
-   image:
-     # highlight-next-line
-     registry: anything
-     # highlight-next-line
-     repository: anything
-     pullPolicy: Always
-     tag: "latest"
-   ```
-
-3. To configure high availability, choose or create a shared file system. For more information, see [Create a shared file system.](/getting-started/setup/installation-guides/kubernetes#create-a-shared-file-system)
-
-4. Add variables related to keycloak to `applicationConfig` section:
-
-   ```yaml
-     APPSMITH_KEYCLOAK_DB_DRIVER: ""
-     APPSMITH_KEYCLOAK_DB_USERNAME: ""
-     APPSMITH_KEYCLOAK_DB_PASSWORD: ""
-     APPSMITH_KEYCLOAK_DB_URL: ""
-     APPSMITH_KEYCLOAK_DB_NAME: "keycloak"
+   helm repo add appsmith-ee https://helm-ee.appsmith.com
    ```
 
-## Install Business Edition Helm chart 
-
-To add and deploy the new Helm chart, run the following command:
+4. Load the Appsmith chart repository with:
 
    ```bash
-   helm repo add appsmith-ee https://helm-ee.appsmith.com
    helm repo update
-   helm install appsmith appsmith-ee/appsmith -n <namespace> -f values.yaml
    ```
 
-For more information, see [installing Business Edition with Kubernetes](/getting-started/setup/installation-guides/kubernetes#install-appsmith).
+5. Deploy Appsmith with:
 
-
-## Restore backup
-
-To restore the backup, follow the below steps:
-
-1. To copy the Appsmith backup into the new Appsmith pod, run the following command:
-
-   ```bash
-   kubectl cp ./<name_of_backup_file> <namespace>/<pod>:/appsmith-stacks/data/backup/
    ```
+   helm install appsmith appsmith-ee/appsmith -n appsmith --create-namespace
+   ```
+
+6. Restore the backup data using the [Restore Appsmith instance](https://docs.appsmith.com/getting-started/setup/instance-management/appsmithctl?current-command-type=kubernetes-commands#restore-instance).
 
-2. To copy the keycloak backup into the new Appsmith pod, run the following command:
+7. Restart the pods with:
 
    ```bash
-   kubectl cp keycloak_bkp.json <namespace>/<pod_name>:/appsmith-stacks/data/
+   kubectl rollout restart statefulsets/appsmith -n <NAMESPACE>
```

Comment:
```suggestion
   kubectl rollout restart statefulsets/appsmith -n appsmith
```

Reply:
As we are using placeholder, keeping this as is.